lib/open_mirego/endpoint.ex

```diff
@@ -1,6 +1,9 @@
 defmodule OpenMirego.Endpoint do
   use Phoenix.Endpoint, otp_app: :open_mirego
 
+  # Always serve requests from a single canonical host
+  plug PlugCanonicalHost, canonical_host: Application.get_env(:open_mirego, OpenMirego.Endpoint)[:url][:host]
```

Comment:
Alright, I’ll create another pull request to use `get_in` everywhere in the application.